db/migrate/20240221113542_create_statements.rb

```diff
@@ -0,0 +1,21 @@
+class CreateStatements < ActiveRecord::Migration[7.1]
+  def change
+    create_enum :statement_states, %w[open payable paid]
+
+    create_table :statements do |t|
+      t.integer :month, null: false
+      t.integer :year, null: false
+      t.date :deadline_date
+      t.date :payment_date
+      t.boolean :output_fee, default: true, null: false
+      t.references :cohort, null: false, foreign_key: true
+      t.references :lead_provider, null: false, foreign_key: true
+      t.datetime :marked_as_paid_at
+      t.decimal :reconcile_amount, precision: 8, scale: 2
+      t.datetime :frozen_at
```

Comment:
What does frozen mean here? Can `open`, `payable` and `paid` statements all be frozen?

Reply:
@peteryates 
`frozen_at` is a new field added to the separation branch: https://github.com/DFE-Digital/npq-registration/commit/4380f4ed1973090f03597aa56d2a4e3c8a4eff18

I'm not sure if we need it, the statement should be "frozen" once its paid.

Reply:
Ah, perhaps @pmanrubia or @cwrw can shed some light.

Reply:
Having a look at the comment here https://github.com/DFE-Digital/npq-registration/pull/1036 basically in ECF currently we use the type "Payable" to allow freezing a statement in our logic. I think we can probably rely on deadline date or the enum state, so probably we can drop this

Reply:
I've removed `frozen_at` for now, if we need it, we can add.